Library/Homebrew/utils/gems.rb

```diff
@@ -61,7 +61,7 @@ def setup_gem_environment!(setup_path: true)
     paths = ENV.fetch("PATH").split(":")
     paths.unshift(ruby_bindir) unless paths.include?(ruby_bindir)
     paths.unshift(Gem.bindir) unless paths.include?(Gem.bindir)
-    paths.unshift(HOMEBREW_SHIMS_PATH/"ruby") unless paths.include?(HOMEBREW_SHIMS_PATH/"ruby")
+    paths.unshift(HOMEBREW_LIBRARY_PATH/"shims/ruby") unless paths.include?(HOMEBREW_LIBRARY_PATH/"shims/ruby")
```

Comment:
What's the motivation here?

Reply:
`HOMEBREW_LIBRARY_PATH` is the only constant that can be used in `utils/gems`, which is one of the first files to load.

I think we got away with it because the `setup_path: true` line is invoked less. It very likely would have broken `HOMEBREW_BOOTSNAP` installs though.

Reply:
Gotcha, works for me 👍🏻